ts/modules/marker-clusters.src.ts

```diff
@@ -288,6 +288,8 @@ var Series = H.Series,
  * @product      highcharts highmaps
  * @since        next
  * @optionparent plotOptions.scatter.cluster
+ *
+ * @private
```

Comment:
Is this related to Annotations?

Reply:
This is related to the class references. The tag gets ignored by the API scripts and just makes sure, that JSDoc is not picking it up itself (to put it in the global namespace).